server/controllers/notes.controller.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express")
+const router = express.Router()
+const notesSvc = require('../services/notes.service')
```

Comment:
Let's not name every file with .service or .controller because they already are in controllers or services folder. Let's not use Svc and similar shortforms and instead use notesService which clearly states what it is.

Reply:
ok

Reply:
done